test/OWNERS

```diff
@@ -44,31 +33,34 @@ approvers:
   - MrHohn
   - deads2k
   - enisoc
-  - enj # for test/integration/etcd/etcd_storage_path_test.go
-  - eparis
-  - erictune
-  - foxish # for test/e2e/network-partition.go
-  - gmarek
   - janetkuo
-  - kow3ns # for test/e2e/statefulset.go
-  - krousey
   - liggitt
-  - madhusudancs
-  - marun
   - mikedanese
-  - msau42 # for test/e2e/commmon/{*volume,empty_dir}.go and test/e2e/framework/{pv|volume}_util.go
+  - msau42
   - ncdc
+  - oomichi
   - pwittrock # for test/e2e/kubectl.go
   - saad-ali
   - shyamjvs
   - sig-testing-approvers
   - smarterclayton
-  - soltysh
```

Comment:
@spiffxp I'm pretty sure I did approve tests over the past 6 months or so, but I don't have the time to look through data. I've opened https://github.com/kubernetes/kubernetes/pull/79170 to add myself back. In the future I'd appreciate ping before doing so.